varcode/effects.py

```diff
@@ -382,13 +395,42 @@ def __init__(
         self.aa_mutation_end_offset = aa_mutation_end_offset
         self.aa_ref = aa_ref
 
-    modifies_protein_sequence = True
+    @property
+    def modifies_protein_sequence(self):
```

Comment:
Oh, the diff is confusingly mixing up the head of `AlternateStartCodon` and the tail of `NonsilentCodingMutation`